ui_framework/components/_common_styles.scss

```diff
@@ -15,3 +15,10 @@
 body {
   font-family: $globalFontFamily;
 }
+
+/**
+ * 1. Required for IE11.
+ */
+main {
+  display: block; /* 1 */
+}
```

Comment:
I created https://github.com/timroes/kibana/pull/1 for now. We can walk through the task when you're done with X-School to figure out what's going on. 😄